integration-test/1491-peruvian-shields.py

```diff
@@ -0,0 +1,135 @@
+# -*- encoding: utf-8 -*-
+from . import FixtureTest
+
+
+class PeruShieldTest(FixtureTest):
+    def test_peayroads(self):
+        import dsl
+
+        z, x, y = (16, 19261, 35238)
+
+        self.generate_fixtures(
+            dsl.is_in('PE', z, x, y),
+            # https://www.openstreetmap.org/way/414535595
+            dsl.way(414535595, dsl.tile_diagonal(z, x, y), {
+                'highway': u'primary',
+                'ref': u'AY-104',
+                'source': u'openstreetmap.org',
+            }),
+            dsl.relation(1, {
+                'network': u'PE-AY-roads',
+                'ref': u'AY-104',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[414535595]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 414535595,
+                'shield_text': '104',
+                'network': u'PE:AY',
+            })
+
+    def test_peroads(self):
+        import dsl
+
+        z, x, y = (16, 19145, 35496)
+
+        self.generate_fixtures(
+            dsl.is_in('PE', z, x, y),
+            # https://www.openstreetmap.org/way/68875825
+            dsl.way(68875825, dsl.tile_diagonal(z, x, y), {
+                'highway': u'trunk',
+                'ref': u'PE-30A',
+                'source': u'openstreetmap.org',
+            }),
+            dsl.relation(1, {
+                'network': u'PE-roads',
+                'ref': u'PE-30A',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[68875825]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 68875825,
+                'shield_text': '30A',
+                'network': u'PE:PE',
```

Comment:
What is the shield_text supposed to read? `30A`?

Reply:
[The shield](https://es.wikipedia.org/wiki/Ruta_nacional_PE-30) looks a bit like a US Interstate one, with just the number (and possibly suffix letter). I think `30A` is the right `shield_text`.